.github/workflows/automated-tests.yml

```diff
@@ -112,74 +112,76 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        shard: [1/8, 2/8, 3/8, 4/8, 5/8, 6/8, 7/8, 8/8]
+        shard: [1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
`upload-artifact@v4` no longer supports multiple exports with the same name. Therefore the shard is now appended to the name. That name must not include a slash, so this change here was required.